python/Dockerfile

```diff
@@ -17,16 +17,12 @@ RUN python --version
 
 WORKDIR /usr/src/app
 
-COPY its-client .
-COPY its-client.cfg .
+COPY its_client its_client
 COPY requirements.txt .
 COPY setup.cfg .
 COPY setup.py .
 
-RUN pip install wheel && \
-    python -m unittest && \
-    python setup.py test && \
```

Comment:
Same question: is there a problem on setup test now? A kind of regression? It worked previously ...

Reply:
Hmm.. Indeed I may have snipped out a bit too much... I'll restore it.

I remember it not working, but it does now... Weird. Probably I fixed something else that made it work again?

Reply:
Ah, now I remember: it failed before I fixed the `COPY its_client .` directive.

But still, it emits a warning:

```
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
```